config/LAGraph.pc.in

```diff
@@ -0,0 +1,18 @@
+# LAGraph, Copyright (c) 2019-2023, Timothy A. Davis.
+# All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: LAGraph
+URL: https://github.com/DrTimothyAldenDavis/SuiteSparse
+Description: Library plus test harness for collecting algorithms that use GraphBLAS in SuiteSparse
```

Comment:
Should references to SuiteSparse be removed when upstreaming this to here?
``` suggestion
URL: https://github.com/GraphBLAS/LAGraph
Description: Library plus test harness for collecting algorithms that use GraphBLAS
```

Reply:
IIUC, the pkg-config format doesn't allow two URLs:
https://manpages.ubuntu.com/manpages/jammy/man5/pc.5.html
>      URL     A URL to a webpage for the package.  This is used to recommend where newer versions
>              of the package can be acquired.  (mandatory; literal)

Reply:
Ok. Then it should be changed and also in suitesparse to match.

Reply:
I also added it to the dev2 branch of SuiteSparse.